### PR TITLE
[DEV-1002] Create mtls and client_secret_basic pocs

### PR DESCRIPTION
**Pull request by yovasx2**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Examples of IDPartner usage without the node-oidc-client library
+
+This repository contains 2 examples:
+1. client-secret-basic-poc: A node project which obtains a token using the client_secret_basic authentication method
```
Comment (chrismccaw) [inline README.md]: I don't like POC in the name. I think better `client-secret-basic-example`

File: README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+   Please check [here](./client-secret-basic-poc/README.md) for further details.
+1. tls-client-auth-poc: A node project which obtains a token using mTLS.
```
Comment (chrismccaw) [inline README.md]: same here

File: client-secret-basic-poc/README.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+## POC for OAuth flow using client_secret_basic
+
+Follow next steps to execute the POC:
+
+1. Run: `yarn install`
+1. Start ngrok pointing to localhost:3001, grab the url: `ngrok 3001`
```
Comment (chrismccaw) [inline client-secret-basic-poc/README.md]: I am thinking of removing ngrok from the documentation. I made sandbox apps not required https. Asking for ngrok seems to confuse people and require more time to setup

File: client-secret-basic-poc/app.js [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+  const queryParams = querystring.stringify({
+    redirect_uri: redirectUri,
+    code_challenge_method: "S256",
+    code_challenge: challenge,
+    state,
+    nonce,
+    scope: "openid offline_access email profile birthdate address",
+    client_id: clientId,
+    client_secret: clientSecret,
+    identity_provider_id: 1, // mikomo
```
Comment (chrismccaw) [inline client-secret-basic-poc/app.js]: why is this needed?

File: tls-client-auth-poc/README.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
+## POC for OAuth flow using mTLS
+
+Follow next steps to execute the POC:
+
+1. Run: `yarn install`
+1. Start ngrok pointing to localhost:3001, grab the url: `ngrok 3001`
+1. Create a tls_cleint_auth app in console.idpartner.com using the ngrok url:
+   origin: <ngrok_url>/button/oauth
+   callback: <ngrok_url>/button/oauth/callback
+1. Create a private key for your applciation using its client_id:
```
Comment (chrismccaw) [inline tls-client-auth-poc/README.md]: typo `applciation`

File: client-secret-basic-poc/README.md (hunk 3 above)
Comment (yovasx2) [reply client-secret-basic-poc/README.md]: but you need ngrok to connect prod with your local

File: client-secret-basic-poc/app.js (hunk 4 above)
Comment (yovasx2) [reply client-secret-basic-poc/app.js]: to complete the oauth flow without using the account selector

File: README.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
+# Examples of IDPartner usage without the node-oidc-client library
+
+This repository contains 2 examples:
+1. client-secret-basic-example: A node project which obtains a token using the client_secret_basic authentication method
+   Please check [here](./client-secret-basic-example/README.md) for further details.
+1. tls-client-auth-example: A node project which obtains a token using mTLS.
```
Comment (chrismccaw) [inline README.md]: should we call this `Mutual TLS`?  not sure devs will understand tls-client-auth

**Review by chrismccaw — APPROVED** [review]

**Pull request merged**